monai/inferers/splitter.py

```diff
@@ -340,10 +336,12 @@ def __init__(
         )
         # Set WSI reader
         self._set_reader(reader, reader_kwargs)
-        if self.pad_mode and self.reader.backend.lower() == "tifffile":
+        if self.reader.backend.lower() == "tifffile":
             raise ValueError(
-                "Padding is not supported with 'TiffFile' reader. "
-                "Please either set the padding to `None` or use 'OpenSlide' or 'cuCIM' reader."
+                "'TiffFile' reader is not appropriate for efficiently loading patches, so it is prohibited. "
+                "Please either use other readers such as 'OpenSlide' and 'cuCIM', or "
+                "first load the whole slide image using `monai.data.WSIReader` with `backend=tifffile` "
+                "(or `monai.data.TiffFileWSIReader`), and then use `SlidingWindowSplitter` instead."
```

Comment:
I see your point here, but there is no reason to disable the TiffFile backend except that it is not efficient, right? Maybe we don't need to disable it here but add more explanation in the doc strings and also add some guide in the tutorial, what do you think? Thanks!

Reply:
I see your point but the problem is if we allow TiffFile here, which is always a wrong thing to do (without exception as fas as I can think), it means that we are supporting it while we don't. However, I can make it less restrictive and raise a warning. How about that?

Reply:
Sure, that would be better. I also want to know if it will be possible that the other two backends are not applicable and only `TiffFile` can be used so that the user will still require this backend even if the speed is slow. Just not sure about that, it would be great if you could help me confirm it, thanks!

Reply:
Basically, use should not use "TiffFile" backend here since it is _unnecessarily_ slow. The reason that I am saying "unnecessarily" is that, the user instead can load the entire WSI into memory first (using TiffFile load it into memory anyways) and use `SlidingWindowSplitter` instead. This simply can give the user a lot faster performance.